### PR TITLE
x86:Fix Model shows "Default String" issue in Luci overview page.

### PR DESCRIPTION
**Pull request by pr0zel**
395865715-0c88ea2e-6bc7-4719-8282-79674a992a1b

after being fixed:
395865774-9c3e2dd4-9ebb-4db9-bff5-6d52a0053860

### CODE REVIEW

File: target/linux/x86/base-files/lib/preinit/01_sysinfo [hunk 1]
```diff
@@ -16,7 +16,9 @@ do_sysinfo_x86() {
 		empty | \
 		System\ manufacturer | \
 		To\ [bB]e\ [fF]illed\ [bB]y\ O\.E\.M\. | \
+		System\ manufacturer | \
```
Comment (namiltd) [inline target/linux/x86/base-files/lib/preinit/01_sysinfo:19]: Twice: System\ manufacturer |

File: target/linux/x86/base-files/lib/preinit/01_sysinfo [hunk 2]
```diff
@@ -29,7 +31,9 @@ do_sysinfo_x86() {
 		?*:empty | \
 		?*:System\ Product\ Name | \
 		?*:To\ [bB]e\ [fF]illed\ [bB]y\ O\.E\.M\. | \
+		?*:System\ manufacturer | \
```
Comment (namiltd) [inline target/linux/x86/base-files/lib/preinit/01_sysinfo:34]: "System manufacturer" will not appear in product name. Line to be removed

**Comment by namiltd** [comment]
Merge everything into one commit.

File: target/linux/x86/base-files/lib/preinit/01_sysinfo (hunk 1 above)
Comment (pr0zel) [reply target/linux/x86/base-files/lib/preinit/01_sysinfo:19]: I made a mistake due to carelessness. I'm wondering if I should close this pull request and submit a new one after fixing the issue. This bug definitely needs to be addressed. As a newbie to the community, I'm a bit lost. Could you please guide me on the next steps? Thank you.

Comment (namiltd) [reply target/linux/x86/base-files/lib/preinit/01_sysinfo:19]:
```
git reset --hard 1aae1dfaf2a1c85aa5bc3faf7e75ff132ab31f95
git cherry-pick -n 95c7f55ef2efe2644546b3fbe3ccfc6b90e3a373
git cherry-pick -n 135cafe9ad9850f20489c0deaa5eca8b55e4b492
```
make additional changes
```
git commit
git push --force
```

Comment (pr0zel) [reply target/linux/x86/base-files/lib/preinit/01_sysinfo:19]: I've got it now. Thank you so much for your detailed explanation and valuable feedback on my code. Your insights have been incredibly helpful. I'll carefully review my changes and resubmit a new PR . Thanks again for all your help!

**Comment by namiltd** [comment]
There is no point in making another thread, reopen this one and post new fixes.

**Pull request closed without merging**